ChineseFilter/sections/antiadblock.txt

```diff
@@ -399,3 +399,6 @@ nfmovies.com#$##adleft { position: absolute!important; left: -3000px!important;
 nfmovies.com#$#img[src^="/pic/tu/"] { visibility: hidden!important; }
 nfmovies.com#$#a[onclick*="openurl"][target="_blank"] img[src^="/static/"] { position: absolute!important; top: -9999px!important; left: -9999px!important; bottom: unset!important; }
 ||nfmovies.com/static/*.jpg$image,redirect=1x1-transparent.gif
+zhangxinxu.com#$#body>*{display:none !important;}
+zhangxinxu.com#$##page,div.main{display:block !important;}
+zhangxinxu.com#$#.hd_remind,[class^="col-left"],[class^="col-right"]{display: none !important; position: fixed !important; top: -9999px !important; opacity: 0 !important;}
```

Comment:
Fixed directly, please take a look how it should be https://github.com/AdguardTeam/AdguardFilters/commit/a4d3e31b96b6b480b4758ecb26d491795dd73d0e